package.json

```diff
@@ -75,7 +75,8 @@
     "yaml": "^2.2.2",
     "semver": "^7.5.2",
     "optionator": "~0.9.3",
-    "typescript": "~5.1.6"
+    "typescript": "~5.1.6",
+    "@cypress/request": "^3.0.0"
```

Comment:
A little confused, the PR title says *bump*, but here you are *adding a new* dep of `@cypress/request`.

If this package was not used before this PR, why was `yarn audit` failing?

Reply:
you need to look in the lockfile, it was previously at 2.88.12 
<img width="861" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/0c9883e7-e526-469f-8c2c-aaed0c111048">